docs/plugins/spectate.rst

```diff
@@ -103,10 +103,22 @@ Settings
     Toggle whether to prefer following (non-siege) units that have newly
     arrived on the map.
 
+``tooltip-follow`` (default: enabled)
+    If the ``spectate.tooltip`` overlay is enabled, toggle whether to show the
+    tooltip.
+
+``tooltip-follow-blink-milliseconds`` (default: 3000)
+    If the ``spectate.tooltip`` overlay is enabled, set tooltip's blink duration
+    in milliseconds. Set to 0 to always show.
+
 ``tooltip-follow-job`` (default: enabled)
     If the ``spectate.tooltip`` overlay is enabled, toggle whether to show the
     job of the dwarf in the tooltip.
 
+``tooltip-follow-job-shortenings``
```

Comment:
I think we need more documentation about the syntax of the `<value>` for this option. How would the player specify the dictionary?

Reply:
added an example to the example section